shared/util/bootstrapable.js

```diff
@@ -3,15 +3,15 @@ import React, {Component} from 'react'
 import {Box, Text} from '../common-adapters'
 import {globalStyles} from '../styles'
 
-export type BootstrapableProp<P> = {
+export type BootstrapableProp<P: Object> = {
   bootstrapDone: false,
   onBootstrap: () => *,
 } | {
   bootstrapDone: true,
   originalProps: P,
 }
 
-export default function Bootstrapable<P> (ComposedComponent: ReactClass<P>): ReactClass<BootstrapableProp<P>> {
+export default function Bootstrapable<P: Object> (ComposedComponent: ReactClass<P>): ReactClass<BootstrapableProp<P>> {
```

Comment:
Flow noticed the spread operator below and inferred that `P` must be an `Object`.